pkgs/development/web/bootstrap-studio/default.nix

```diff
@@ -0,0 +1,36 @@
+{ lib, fetchurl, appimageTools }:
+
+let
+  pname = "bootstrap-studio";
+  version = "6.0.1";
+  name = "bootstrap-studio-${version}";
+  nameExecutable = pname;
+  src = fetchurl {
+    url = "https://bootstrapstudio.io/releases/desktop/${version}/Bootstrap%20Studio.AppImage";
+    sha256 = "sha256-piRqIB/bCF0IBZfXdlXrc7gmPTIUDbk8xhP7X5ozyWg=";
+  };
+  appimageContents = appimageTools.extractType2 { inherit name src; };
+in
+appimageTools.wrapType2 {
+  inherit name src;
```

Comment:
```suggestion
  inherit pname version src;
```

Reply:
I'm not how that would work, wrapType2 only takes a name, src, and extraPkgs?
![image](https://user-images.githubusercontent.com/44947946/166918488-4e087eb7-562c-4d9a-8b5c-4829930442eb.png)
[Source](https://github.com/NixOS/nixpkgs/blob/master/pkgs/build-support/appimage/default.nix)

Reply:
Ah I see, I missed the default value for name -- thank you, I've committed this